docs/backup.md

```diff
@@ -0,0 +1,112 @@
+# Backup
+
+Follow this guide to create backup for elemental configuration installed together with Rancher.
+
+## Install rancher-backup operator for Rancher
+
+Go to official [Rancher documentation](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/back-up-rancher) and install rancher-bakup operator from there.
+
+!!! warning
+    For Rancher v2.6 and below it is needed to edit ResourceSet for rancher-backup operator.
+    For Rancher v2.7 and above backup will be done automatically by rancher-backup operator and no further operation are needed.
```

Comment:
```suggestion
    For Rancher v2.7.1 and above backup will be done automatically by rancher-backup operator and no further operation are needed.
```

Reply:
because rancher/backup-restore-operator/pull/277 will be late :-/

Reply:
updated